src/app/App.module.css

```diff
@@ -0,0 +1,32 @@
+.container {
+  display: grid;
```

Comment:
I would suggest adding a small gap here in order to show the lower border of your submit button correctly (I hope that works...)